server/routes/homepage/homepage.ts

```diff
@@ -38,6 +38,15 @@ export const tasks: TaskType[] = [
     roles: ['REMOVE_RESTRICTED_PATIENT'],
     enabled: true,
   },
+  {
+    id: 'migrate-into-hospital',
+    heading: 'Migrate released prisoner into hospital',
+    description:
+      'Migrate a released prisoner into the restricted patients service.  This amend a prisoner record who has previously been released using NOMIS to be a restricted patient and alters their last movement to include the hospital location.',
```

Comment:
> 'This amend a prisoner record who has previously been released using NOMIS to be a restricted patient and alters their last movement to include the hospital location.'

This doesn't make complete sense to me, this could be because of lack of context.
Perhaps 'This amends a prisoner's record'